src/cv2pdb.cpp

```diff
@@ -166,7 +166,11 @@ bool CV2PDB::openPDB(const TCHAR* pdbname, const TCHAR* pdbref)
 	printf("PDB::QueryPdbImplementationVersion() = %d\n", pdb->QueryPdbImplementationVersion());
 #endif
 
-	rsds = (OMFSignatureRSDS *) new char[24 + strlen(pdbnameA) + 1]; // sizeof(OMFSignatureRSDS) without name
+	rsdsLen = 24 + strlen(pdbnameA) + 1; // sizeof(OMFSignatureRSDS) without name
+	// Growing the RSDS block to the closest 16-byte boundary.
+	if (rsdsLen & 0xF)
+		rsdsLen = (rsdsLen | 0xF) + 1;
+	rsds = (OMFSignatureRSDS *) new char[rsdsLen];

```

Comment:
The padding memory should be initialized to zero, so just `memset` the whole chunk.

Reply:
Ok.